### DISCUSSION TITLE
第4节课笔记(18班)

### DISCUSSION THREAD

**Discussion by mattheliu** [InternLM/Tutorial#148] [笔记]
请提交笔记链接
Please submit the link to the notes.

**Comment by scorpionfree98** [comment]
[书生 浦语大模型实战营 - 笔记&作业4 大模型单卡微调](http://t.csdnimg.cn/E5TXL)

**Comment by 2001wjh** [comment]
https://www.yuque.com/wjh2001/rdvfen/gkrlvc9q4y1v1s8g?singleDoc# 《第4节课笔记》

**Comment by DrZhang1208** [comment]
[第四节课作业：XTuner 大模型单卡低成本微调实战](https://blog.csdn.net/qq_42866033/article/details/135557038?spm=1001.2014.3001.5501)

**Comment by ZMcc18** [comment]
https://www.yuque.com/zixul/avxnzn/yfsm2dkf4gaxx2y0?singleDoc# 《lesson4 XTuner 大模型单卡低成本微调实战 笔记》

**Comment by Heitao5200** [comment]
https://github.com/Heitao5200/my_blog/blob/master/docs/%E7%BB%84%E9%98%9F%E5%AD%A6%E4%B9%A0/%E5%A4%A7%E6%A8%A1%E5%9E%8B%E5%AE%9E%E6%88%98/TASK4.md

**Comment by yestolife** [comment]
[笔记+作业：Xtuner大模型微调](http://t.csdnimg.cn/1UvwZ)

**Comment by tackhwa** [comment]
[第4节课笔记+作业](https://nutritious-chill-7a9.notion.site/XTuner-2e9c13f9671043b7bee24df3b22d8979?pvs=4）

**Comment by ggfengxuan** [comment]
[4.XTuner大模型单卡低成本微调实战笔记](https://v1y50l9n58a.feishu.cn/wiki/DngpwAu2niah9pkl4bucdITYnNc?from=from_copylink)

**Comment by myaijarvis** [comment]
【参考：[【笔记】书生·浦语大模型实战营——第四课（XTuner 大模型单卡低成本微调实战）-CSDN博客](https://blog.csdn.net/Jruo911/article/details/135585579 )】

**Comment by xwhclaire** [comment]
[Lecture 4 XTuner大模型单卡低成本微调实战](https://github.com/xwhclaire/StudyPackages/blob/main/LLM%20training/Lecture%204%20XTuner%E5%A4%A7%E6%A8%A1%E5%9E%8B%E5%8D%95%E5%8D%A1%E4%BD%8E%E6%88%90%E6%9C%AC%E5%BE%AE%E8%B0%83%E5%AE%9E%E6%88%98.md)

**Comment by mattheliu** [comment]
https://github.com/mattheliu/gitblog/issues/4
